src/sentry/migrations/0950_fix_span_item_event_type_alerts.py

```diff
@@ -0,0 +1,87 @@
+# Generated by Django 5.2.1 on 2025-07-15 15:44
+
+from enum import Enum
+from typing import Any
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class EventType(Enum):
+    ERROR = 0
+    DEFAULT = 1
+    TRANSACTION = 2
+    TRACE_ITEM_SPAN = 3
+    TRACE_ITEM_LOG = 4
+
+
+def fix_span_item_event_type_alerts(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    SnubaQuery = apps.get_model("sentry", "SnubaQuery")
+    SnubaQueryEventType = apps.get_model("sentry", "SnubaQueryEventType")
+
+    for snuba_query in RangeQuerySetWrapperWithProgressBar(
+        SnubaQuery.objects.filter(dataset="events_analytics_platform")
+    ):
```

Comment:
select count(*) from sentry_snubaquery where dataset='events_analytics_platform' - it's a few hundred rows